drivers/sensor/omron_2smpp03/omron_2smpp03.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2020 Teslabs Engineering S.L.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/adc.h>
+#include <drivers/sensor.h>
+#include <devicetree/adc.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(omron_2smpp03, CONFIG_SENSOR_LOG_LEVEL);
+
+/** ADC gain. */
+#define ADC_GAIN ADC_GAIN_1
+/** ADC reference. */
+#define ADC_REF ADC_REF_INTERNAL
+/** ADC acquisition time. */
+#define ADC_ACQT ADC_ACQ_TIME_DEFAULT
+/** ADC resolution. */
+#define ADC_RES CONFIG_OMRON_2SMPP03_ADC_RESOLUTION
+
+/** 2SMPP03 offset (mV) */
+#define SENSOR_OFFSET -2.5f
+/** 2SMPP03 gain (kPa/mV) - 50kPa: 42mV, -50kPa: -43mV. */
+#define SENSOR_GAIN (100.0f / 85.0f)
```

Comment:
Can you rework the driver to not using floating point calculations?

Reply:
Done, thanks for this suggestion!